vars/withPipeline.groovy

```diff
@@ -127,6 +127,17 @@ def call(type, String product, String component, Closure body) {
             product: product,
           )
 
+          sectionPromoteBuildToStage(
+            appPipelineConfig: pipelineConfig,
+            pipelineCallbacksRunner: callbacksRunner,
+            pipelineType: pipelineType,
+            subscription: subscription.nonProdName,
+            product: product,
+            component: component,
+            stage: DockerImage.DeploymentStage.PREVIEW,
+            environment: environment.previewName
+          )
```

Comment:
what does this end up doing? does it create a latest tag for preview? i.e. pr-111

Reply:
It does yeah

Reply:
shouldn't that be done after `sectionDeployToAKS`?

Reply:
Section was added to test in toffee with a dedicated stage. Was to be removed before merging - gone now